src/utils/ExtractTestResults.ts

```diff
@@ -69,8 +72,8 @@ export const extractMCTestResults = (record: DynamoDBRecord): MCRequest[] => {
  * This method is used to change the test result to be a single, uppercase character
  * @param testResult
  */
-export const calculateTestResult = (testResult: TestResult): string =>
-  testResult.testResult.toLowerCase() === 'pass' ? 'S' : 'R';
+export const calculateTestResult = (testResult: TestResults): string =>
+  testResult === TestResults.PASS ? 'S' : 'R';
```

Comment:
```suggestion
  testResult.toLowerCase() === TestResults.PASS ? 'S' : 'R';
```
I'd suggest using the to uppercase on this, so that we can ensure that it's always lowercase. Appreciate that it's very unlikely that it'll be upper case though

Reply:
do you mean lowercase?

With testResult type being a hardcoded value from typedef I thought it an unnecessary step. a pass testResult will always === TestResults.PASS surely?

Reply:
I did mean lowercase sorry. It would but I don't think it wouldn't hurt to have the lowercase functionality there, just in case

Reply:
I see the argument. might be good for it to error if they ever dont match though, as then we'll be able to catch and fix the type mismatch.